dev_docs/cxg.md

```diff
@@ -73,38 +70,38 @@ For example:
+Optional TileDB array, used to encode per-column offsets for column-shift sparse encoding.  The TileDB array will have a single anonymous attribute, of the same type as the X array, and shape `(n_var,)`.  
 
-If the X array is sparse, and X_col_shift exists, then all values in the i'th column were subtracted by X_col_shift[i].
+If the `X` array is sparse, and the `X_col_shift` array exists, then all values in the `i`'th column were subtracted by `X_col_shift[i]`.
 
-### emb and embedding arrays
+### `emb` group and embedding arrays
 
-A CXG must have a group named `emb`, which will contain all embeddings.  Embeddings are encoded as TileDB arrays, of numeric type and shape (n_obs, >=2).  The arrays SHOULD be coerced to float32, and MUST be a numeric type.  The TileDB array name will be assumed to be the embedding name (conventionally, embedding names in CXG are _not_ prefixed with an `X_` as they are in AnnData).
+A CXG MUST have a group named `emb`, which will contain all embeddings.  Embeddings are encoded as TileDB arrays, of numeric type and shape `(n_obs, >=2)`.  The arrays MUST be a numeric type, and SHOULD be coerced to `float32`. The array values MUST NOT contain any infinite values and MUST contain at least one non-NaN value.  The TileDB array name will be assumed to be the embedding name (conventionally, embedding names in CXG are _not_ prefixed with an `X_` as they are in AnnData).
 
 CXG supports zero or more embeddings. Note that cellxgene currently _requires_ at least one embedding.
 
-### cxg_group_metadata
+### `cxg_group_metadata` group
 
 Required, but empty TileDB array, used to store CXG-wide metadata.  The following fields are defined:
-* __cxg_version__: (required) a semver string identifying the specification version used to encode the CXG.
-* __cxg_properties__: (optional) a dictionary containing dataset wide properties, defined below.
-* __cxg_category_colors__: (optional) a categorical color table, defined below.
+* `__cxg_version__`: (required) a [semantic version](https://semver.org/) identifying the specification version used to encode the CXG.
+* `__cxg_properties__`: (optional) a dictionary containing dataset wide properties, defined below.
+* `__cxg_category_colors__`: (optional) a categorical color table, defined below.
 
-#### cxg_properties
+#### `cxg_properties` field
 
-The properties metadata dictionary contains dataset-wide properties, encoded as a JSON dictionary. Currently, the following fields are defined:
-* title: string, dataset human name (eg, "Lung Tissue")
-* about: string, fully-qualified http/https URL, linking to more information on the dataset.
+The `__cxg_properties__` metadata dictionary contains dataset-wide properties, encoded as a JSON dictionary. Currently, the following fields are defined:
```

Comment:
properties of this metadata field should be noted as optional/mandatory.  Currently, I  believe they are all optional (eg, you can have a cxg_properties dict that only contains 'title')

Reply:
now noted all are optional (verified in explorer code this is safe)